### ISSUE TITLE
(low priority) shouldn't body.header not scroll with the sections?

### ISSUE THREAD

**Issue by ghost** [Jasonette/JASONETTE-iOS#346]
The docs seems to imply that the sections are scrollable but the body.header should not. With a templates/body/header containing a search box scrolling the templates/body/sections[0]/items causes the search box also to scroll. 

If this is as-designed then please just close, if this strikes you as not what's intended then I will provide a jasonbase test case from my app.

**Comment by gliechtenstein** [comment]
Yes that's because that's how native search bar works on iOS. One workaround is to move this into the top header bar as you mentioned, but this will give rise to another problem, which is "where to put the title" if the search bar is placed in the top header bar by default? This would mean all apps with a search feature will not be able to have the top title label.

Android doesn't have this problem because their title is by default displayed on the left side, but on iOS I think we'll need to a solution to this problem if we want to support this feature.
